Makefile

```diff
@@ -151,6 +163,10 @@ config-generated.go:
 
 $(TARGET): $(SOURCES) $(GENERATED_FILES) Makefile | show-summary
 	$(QUIET_BUILD)go build -i -o $@ .
+	$(QUIET_BUILD)go build -i -ldflags "-X main.commit=${COMMIT} -X main.version=${VERSION} -X main.libExecDir=${LIBEXECDIR}" -o $@ .
+
+pause: $(PAUSEDIR)/$(PAUSEBIN).o
```

Comment:
Well, right now the way we do pause is:
`make`
`make pause`
`make install`

if we add it by default to the make, we also need to change the .ci/  as it is getting and making pause from `virtcontainers`

https://github.com/clearcontainers/tests/blob/master/.ci/install_virtcontainers.sh

And probably qa needs to  change something as well.. 

@chavafg @devimc need feedback from you here..